nexus/reconfigurator/execution/src/dns.rs

```diff
@@ -1067,7 +1099,9 @@ mod test {
                 .map(|record| match record {
                     DnsRecord::A(v) => IpAddr::V4(*v),
                     DnsRecord::Aaaa(v) => IpAddr::V6(*v),
-                    DnsRecord::Srv(_) => panic!("unexpected SRV record"),
+                    other => {
```

Comment:
nit, take it or leave it: I'd explicitly put the three patterns here.  That will prompt someone to revisit this if we add future ones.